apps/inspector/detectors.py

```diff
@@ -0,0 +1,1129 @@
+    for idx, issue in enumerate(typo_issues):
+        typo_data.append(
+            {
+                "index": idx,
+                "typo": issue.get("typo", ""),
+                "correction": issue.get("correction", ""),
+                "context": issue.get("context", "")[:300],
+            }
+        )
+
+    semantic_data = []
+    for idx, issue in enumerate(other_issues):
+        semantic_data.append(
+            {
+                "index": idx,
+                "type": issue.get("issue_type"),
+                "explanation": issue.get("explanation", "")[:200],
+            }
+        )
+
+    # Single combined prompt for grouping and pruning issues
+    prompt = f"""You are reviewing issues found in OWID (Our World in Data) content. Your goal: REMOVE low-value issues from the list, KEEP only issues that pose reputational risk.
+
+TASK 1: GROUP SIMILAR ISSUES
+- Typos: Group by identical misspelling + correction pair
+- Semantic issues: Group by identical problem description (across different views)
+
+INPUT DATA:
+
+Typos ({len(typo_data)} issues):
+{json.dumps(typo_data, indent=2) if typo_data else "[]"}
+
+Semantic issues ({len(semantic_data)} issues):
+{json.dumps(semantic_data, indent=2) if semantic_data else "[]"}
+
+TASK 2: DECIDE WHICH ISSUE GROUPS TO REMOVE
+
+Review each group's context. REMOVE groups from the list if they are FALSE POSITIVES or LOW-VALUE:
+
+TYPO GROUPS - REMOVE from list if:
+- NOT actually typos: The flagged "typo" is correct in context (scientific names like "Escherichia coli", technical terms like "eigenvalue", proper nouns like "McDonald")
+- Codespell false positives that are actually correct spellings
+
+TYPO GROUPS - KEEP in list if:
+- Real typos: Actual misspellings like "teh" → "the", "recieve" → "receive"
+
+SEMANTIC ISSUE GROUPS - REMOVE from list if:
+- Minor style suggestions that don't affect meaning
+- Tiny inconsistencies (capitalization preferences, punctuation style)
+- Subjective writing preferences
+
+SEMANTIC ISSUE GROUPS - KEEP in list (reputational risk):
+- Inappropriate/offensive language (bad words, slurs, profanity) - CRITICAL: ALWAYS KEEP
+- Factual errors or contradictions
+- Major inconsistencies that confuse readers
+- Unclear or ambiguous statements
+- Anything that could damage OWID's reputation or credibility
+
+OUTPUT FORMAT:
+
+Return ONLY valid JSON:
+{{
+  "typo_groups": {{"group_name": [0, 1, 2]}},
+  "semantic_groups": {{"group_name": [0, 1, 2]}},
+  "spurious_typo_groups": ["group_name1"],
+  "spurious_semantic_groups": ["group_name2"]
+}}
+
+CRITICAL: When unsure about severity, KEEP the issue in the list. Missing a reputational risk is worse than a false positive.
+Write arrays explicitly [0, 1, 2], NOT Python code."""
+
+    try:
+        client = anthropic.Anthropic(api_key=api_key)
+        response = call_claude(
+            client=client,
+            model=GROUPING_MODEL,  # Use higher-quality model for grouping/pruning
+            max_tokens=3072,
```

Comment:
i think this might be hardcoded. I think you had a variable defined under `inspector/config`